app/javascript/app/styles/themes/card/card-light.scss

```diff
@@ -4,8 +4,8 @@
   color: "pink";
   border-radius: 6px;
   border-color: transparent;
-  box-shadow: 0 2px 20px 0 #E8ECF5;
-  margin: 0.9375rem;
+  box-shadow: 0 2px 20px 0 #e8ecf5;
```

Comment:
It would be nice to use some color variable that we have on settings or a box-shadow variable. Maybe this is a new box-shadow standard so we can add a variable for this

Reply:
Hey @Bluesmile82, I added two new colors to `settings.scss` file, I think for now there is no need for adding `box-shadow` as a constant because it seems to be the only place that we're going to use that :)

Reply:
So I think I prefer adding a box-shadow constant than new colors. I don't know if this colors are used somewhere else. This box-shadow will probably be used somewhere else when we change the designs

Reply:
Maybe we need the color constants too. I'm just afraid of too many colors in the settings. I have Polychromofobia

Reply:
I didn't find this box-shadow on Invision in other places, nor in the project. Do you think it's still worth it to put it there?

Reply:
Ok, let's leave it as it is then